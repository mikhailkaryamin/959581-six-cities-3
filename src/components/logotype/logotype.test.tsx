import React from "react";
import renderer from "react-test-renderer";
import {
  BrowserRouter
} from 'react-router-dom';
import Logotype from "./logotype";

it(`Render logo correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <Logotype />
        </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
