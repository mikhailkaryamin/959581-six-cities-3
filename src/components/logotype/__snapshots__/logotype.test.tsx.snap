// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render logo correctly 1`] = `
<a
  className="header__logo-link header__logo-link--active"
  href="/"
  onClick={[Function]}
>
  <img
    alt="6 cities logo"
    className="header__logo"
    height="41"
    src="undefined/img/logo.svg"
    width="81"
  />
</a>
`;
